src/module.rs

```diff
@@ -284,6 +284,8 @@ impl Module {
   /// Returns an empty Maybe<bool> if an exception occurred during
   /// instantiation. (In the case where the callback throws an exception, that
   /// exception is propagated.)
+  ///
+  /// NOTE: requires to set `--harmony-import-assertions` V8 flag.
```

Comment:
Calling this at all need the flag?

Reply:
Yes it does. I might have made a mistake removing old methods, could have left them with `deprecated_` prefix.

Reply:
Ok.

Well - we apologize to any onlookers for the bumpy road.